### PR TITLE
Add migrate command to gaiad

### PR DESCRIPTION
**Pull request by sabau**
Add migrate command in gaiad

`gaiad migrate [version] [genesis] > [new_genesis]`

- need to update cosmos-sdk in go.sum once https://github.com/cosmos/cosmos-sdk/pull/4471 in cosmos sdk is merged

Signed-off-by: Karoly Albert Szabo <szabo.karoly.a@gmail.com>

- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry: `clog add [section] [stanza] [message]`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Review by alexanderbez — CHANGES_REQUESTED** [review]
Doesn't this need the new SDK version in `go.mod`?

**Comment by sabau** [comment]
Yeah I was waiting for the Migrate command to be merged before updating the `go.mod` here

**Comment by sabau** [comment]
https://github.com/cosmos/gaia/pull/56 should land first, fixes all the problems in this branch

**Review by alexanderbez — APPROVED** [review]
ACK

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module github.com/cosmos/gaia
 go 1.12
 
 require (
-	github.com/btcsuite/btcd v0.0.0-20190629003639-c26ffa870fd8 // indirect
+	github.com/btcsuite/btcd v0.0.0-20190523000118-16327141da8c // indirect
```
Comment (alessio) [inline go.mod:6]: This seems a sort of downgrade. I'd like to investigate further before approving this

**Review by alessio — CHANGES_REQUESTED** [review]
There's a minor and weird thing that might be worth of extra investigation

**Pull request merged**